moveit_kinematics/ikfast_kinematics_plugin/templates/ikfast61_moveit_plugin_template.cpp

```diff
@@ -41,13 +41,30 @@
  * This file, including the ikfast cpp from OpenRAVE below, forms a MoveIt kinematics plugin.
  */
 
-#include <ros/ros.h>
+#include <rclcpp/rclcpp.hpp>
 #include <moveit/kinematics_base/kinematics_base.h>
 #include <moveit/robot_state/robot_state.h>
 #include <Eigen/Geometry>
 #include <tf2_kdl/tf2_kdl.h>
 #include <tf2_eigen/tf2_eigen.h>
-#include <eigen_conversions/eigen_kdl.h>
+
+// TODO(JafarAbdi): Copied from eigen_conversions/eigen_kdl
+namespace tf
+{
+template <typename T>
+void transformEigenToKDLImpl(const T& e, KDL::Frame& k)
+{
+  for (unsigned int i = 0; i < 3; ++i)
+    k.p[i] = e(i, 3);
+  for (unsigned int i = 0; i < 9; ++i)
+    k.M.data[i] = e(i / 3, i % 3);
+}
+
+void transformEigenToKDL(const Eigen::Isometry3d& e, KDL::Frame& k)
```

Comment:
Do we need this template specification? I would remove it and simply call the template `transformEigenToKDL`